proto/regen/ecocredit/marketplace/v1/tx.proto

```diff
@@ -25,8 +25,8 @@ service Msg {
   // Buy creates credit buy orders.
   rpc Buy(MsgBuy) returns (MsgBuyResponse);
 
-  // AllowAskDenom is a governance operation which authorizes a new ask denom to
-  // be used in sell orders
+  // AllowAskDenom is a governance operation which authorizes new ask denoms and/or removes denoms to
+  // be used in sell orders.
   rpc AllowAskDenom(MsgAllowAskDenom) returns (MsgAllowAskDenomResponse);
```

Comment:
I think the original intention here was to add an ask denom, i.e. "to allow it". We should probably have two separate methods, one for adding and one for removing and we should only allow one at a time. I think we should avoid batch updates to governance params and each addition/removal warrants its own governance proposal.

Reply:
yeah i think it makes sense to remove batching. governance can do batch updates if needed anyway, as the new v0.46 gov updates allows for multiple messages to execute from a single proposal

Reply:
TBH, I like the idea of packing this into one proposal (as it's done now). Imagine there are 10 denoms, and you want to change them to 10 other denoms. That would require 20 proposals. 
At the same time, if there was no clarity during the signaling, one can still do small changes using a proposal with only one edit.